CMS/settings/base.py

```diff
@@ -206,6 +206,7 @@
 SEARCHAPIHOST = os.environ.get('SEARCHAPIHOST')
 DATASETAPIHOST = os.environ.get('DATASETAPIHOST')
 WIDGETAPIKEY = os.environ.get('WIDGETAPIKEY')
+WIDGETAPIHOST = os.environ.get('WIDGETAPIHOST', os.environ.get('DATASETAPIKEY'))
```

Comment:
Firstly did you mean to set the `DATASETAPIKEY` to be the `WIDGETAPIHOST` if `WIDGETAPIHOST` is not set?
I think you want `DATASETAPIHOST`?

Secondly the `WIDGETAPIKEY` value can not access the dataset API, so if the WIDGETAPIKEY is set and the WIDGETAPIHOST is not set then it will still fail to get a response from the widget API.

What you can do is use the dataset API key to access the widget API - I'd prefer you did not do this as we would have to change it later.

Reply:
@nshumoogum I've removed the fallback.